### PR TITLE
Setup: Initial Setup and Build files

### PR DESCRIPTION
**Pull request by dikshamodi**
Initial setup files: This file contains the initial setup files for an empty spring app and the build files for cloud and app engine.

### CODE REVIEW

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+/usr/local/google/home/dikshamodi/hyperlocal-gpay-spot/src/main/java/com/example/hyperlocal/MerchantController.java
```
Comment (harshil93) [inline target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst:1]: Are these autogenerated ? If yes, put them in .gitignore.

Same for .class files

File: target/classes/static/index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  </body>
+
+  <script>
+    const request = {nonce: 'ahskid'};
+    microapps.getIdentity(request).then(response => {
+      const decodedIdentity = JSON.parse(atob(response.split('.')[1]));
+      console.log("getIdentityResponse", decodedIdentity);
+    }).catch(error => {
+      console.error("Error happened: ", error);
+    });
```
Comment (harshil93) [inline target/classes/static/index.html:73]: Is this code working or required ?

File: target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst (hunk 1 above)
Comment (dikshamodi) [reply target/maven-status/maven-compiler-plugin/compile/default-compile/inputFiles.lst]: I've removed the autogenerated and binary files

File: target/classes/static/index.html (hunk 2 above)
Comment (dikshamodi) [reply target/classes/static/index.html]: Ok. Removed that too. Changed it to a simple index.html file

**Pull request merged**